test/runtime/runtime-kotlin/src/test/kotlin/MyClassTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.mycompany.app
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class MyClassTest {
+    @Test
+    fun testPrintMessage() {
+        val myClass = MyClass()
+        val name = "John"
+        val expectedMessage = "Hello, $name"
+        myClass.printMessage(name)
```

Comment:
After you change the generation script, you need to change this test setup a bit.